docs/EMERGENCY.md

```diff
@@ -0,0 +1,213 @@
+- Does the solution need a longer term plan, can team identified owners for tasks/steps towards execution of said plan?
+
+8. Once a solution has been implemented, the team will validate the solution fixes the cause of the issue and minimizes the loss of funds as much as possible. Possible actions needed during this step:
+
+- Run in ganache fork simulations of end state to confirm solutions
+- Coordinate signatures from multi-sig signers and execution
+- Enable UI changes to normalize operations as needed
+
+9. Assign ownership for redacting [disclosure](https://github.com/yearn/yearn-security) and collection of timeline and events.
+
+10. Team Quorum can decide if the War Room can end. Is up to the facilitator to break down the War Room and set reminders if this takes longer than a few hours for members to reconvene.
+
+### Emergency Checklist
+
+This checklist should be complemented with the [steps](#emergency-steps)
+
+- War room with audio created
+- Key roles are covered in the War Room
+- Strategist or expert related to issue is available or a backup instead
+- Is Multi sig queue clear or actions started to clear it
+- Issue has been confirmed and identified
+- Immediate corrective/preventive actions are put in place and risk of loss of funds prevented
+- Communications of current situation
+- Research causes
+- Brainstorm solutions
+- Implement and validate solutions
+- Prioritize solutions
+- Team Quorum on solution reached
+- Coordination and Execution of solution
+- Monitoring and incident resolution confirmed
+- Assign Disclosure Task and Owner to collect events and timelines
+- Disband War Room
+
+### Tools
+
+List of tools and alternatives in case primary tools are not available during incident.
+
+| Description         | Primary                                        |                               Secondary                                |
+| ------------------- | ---------------------------------------------- | :--------------------------------------------------------------------: |
+| Code Sharing        | Github                                         | [HackMd](https://hackmd.io/), [CodeShare](https://codeshare.io/5Og7mj) |
+| Communications      | Telegram                                       |                                Discord                                 |
+| Transaction Details | [Etherscan](https://etherscan.io/)             |                    [EthTxInfo](https://ethtx.info/)                    |
+| Debugging           | Brownie                                        |                    [Tenderly](https://tenderly.co/)                    |
+| Transaction Builder | [ape-safe](https://github.com/banteg/ape-safe) |               Backup if gnosis safe Api is not working??               |
+| Screen Sharing      | [jitsi](https://jitsi.org/)                    |                            Google Hangouts                             |
```

Comment:
I think anyone can join a jitsi if they know the link. Care should be taken to ensure that only War Room participants can join.